config/lib/express.js

```diff
@@ -114,6 +114,7 @@ module.exports.initSession = function (app, db) {
   app.use(session({
     saveUninitialized: true,
     resave: true,
+    maxAge: config.sessionExpiration,
```

Comment:
I'm not sure, but shouldn't this be:  `cookie: {maxAge: config.sessionExpiration},` instead?